app/controllers/sessions_controller.rb

```diff
@@ -3,18 +3,19 @@ def new
   end
 
   def create
-    user = User.find_by(email: params[:session][:email].downcase)
-    if user&.authenticate(params[:session][:password])
-      log_in user
-      redirect_to user_url(user)
+    @user = User.find_by(email: params[:session][:email].downcase)
+    if @user&.authenticate(params[:session][:password])
```

Comment:
[Q]
do you know why we use `&` in this case ?

Reply:
[A] This is Safe Navigation operator. I think this is a short way to check a nil variable before doing anything with it.